terraform/031-email-service/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output "hostname" {
+  value = "${cloudflare_record.emaildns.hostname}"
+}
+
+output "db_emailservice_user" {
```

Comment:
I don't remember... since this isn't a value generated within this module, why is this an output?

Reply:
not sure, cant think of why we would need that outside the module